tracelib/trace.js

```diff
@@ -149,18 +155,19 @@ function init() {
           // Use start time of the first trace as timestamp to for flow start,
           // to display a range encompassing all duration events in devtools.
           flow = module.exports.flow(Object.assign({ts: trace.ts}, baseInfo)).start();
+        } else {
+          flow.step(baseInfo);
         }
         trace.end(info);
         trace = null;
         try {
           return await v;
         } finally {
-          trace = module.exports.start(baseInfo);
-          flow.step(baseInfo);
```

Comment:
I think this changes the behavior of where the flow arrow terminates.

Reply:
Updated to start the flow at the end of the completion event, while ending arrows at the beginning of the following completion event.